jwst/tweakreg/tweakreg_step.py

```diff
@@ -322,9 +256,15 @@ def process(self, input):
                 yoffset=self.yoffset
             )
 
+            # FIXME can we make these error specific to avoid needing to check
+            # the error message?
             try:
+                # FIXME `align_wcs` checks that all items in `correctors` are
+                # instances of `WCSCorrector` it them makes a new list
+                # with the catalogs. `align_wcs` changes would be needed to
+                # avoid loading all `correctors` into memory
                 align_wcs(
```

Comment:
`align_wcs` checks that all items in `correctors` are instances of `WCSCorrector` it them makes a new list with the catalogs. `align_wcs` changes would be needed to avoid loading all `correctors` into memory. As each `corrector` contains the input image wcs and catalog (and with this PR no longer keeps a reference to the input model) it's doubtful that keeping all these in memory is a large burden.